Asterix/wfsc/corrector.py

```diff
@@ -7,12 +7,13 @@
 import numpy as np
 from astropy.io import fits
 
-import Asterix.save_and_read as useful
-import Asterix.optical_systems as OptSy
-from Asterix.estimator import Estimator
-import Asterix.WSC_functions as wsc
-from Asterix.THD_quick_invert import THD_quick_invert
+from .estimator import Estimator
 
```

Comment:
Unnecessary blank line.
Also, I see that sometimes you use absolute imports (`from Asterix.blah import this`) and sometimes relative imports like here (`from .blah import this`). Is there a reason for this?

Reply:
here we are in the same folder, we're not in Asterix root. correcged the import order

Reply:
corrected